run_under_valgrind.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
It's useful to use the `env` program instead of do just `#/bin/sh`. See https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash.